spec/requests/api/my_request/requests/create_spec.rb

```diff
@@ -108,40 +106,44 @@
       end
     end
 
-    describe 'with valid credentials and sevral missing params' do
+    describe 'user dont have enough karma_points' do
       before do
         post '/api/my_request/requests',
              headers: headers,
              params: {
                title: 'reQuestus title',
-               description: 'You shall come and help me!'
+               description: 'You shall come and help me!',
+               reward: 200
              }
       end
       it 'has 422 response' do
         expect(response).to have_http_status 422
       end
 
       it 'responds with an error message' do
-        expect(response_json['message']).to eq "Reward can't be blank and Reward is not a number"
+        expect(response_json['message']).to eq 'You dont have enough karma points'
       end
     end
 
-    describe 'user dont have enough karma_points' do
+    describe 'when trying to set an invalid category' do
```

Comment:
we had a copy of this one, that's why it looks like I replaced it.

Reply:
It's overriding a lot of the responses we are otherwise testing. I think the tests should be modified to allow the underlying messages to come through.